src/Forms/FieldList.php

```diff
@@ -183,11 +172,9 @@ protected function flushFieldsCache()
      * @param string $tabName The name of the tab or tabset.  Subtabs can be referred to as TabSet.Tab
      *                        or TabSet.Tab.Subtab. This function will create any missing tabs.
      * @param FormField $field The {@link FormField} object to add to the end of that tab.
-     * @param string $insertBefore The name of the field to insert before.  Optional.
-     *
-     * @return $this
+     * @param string|null $insertBefore The name of the field to insert before.  Optional.
```

Comment:
```suggestion
     * @param string|null $insertBefore The name of the field to insert before.
```
No need to say it's optional - that's clear by the fact that it's nullable. `addFieldsToTab` doesn't say "optional" so either remove from here or add to there.

Reply:
Done